cylc/flow/install_plugins/log_vc_info.py

```diff
@@ -162,14 +163,14 @@ def get_vc_info(path: Union[Path, str]) -> Optional[Dict[str, Any]]:
             missing_base = True
             LOG.debug(exc)
         except OSError as exc:
-            if not any(
+            if any(
                 exc.strerror.lower().startswith(err)
                 for err in NOT_REPO_ERRS[vcs]
             ):
-                raise exc
-            else:
                 LOG.debug(f"Source dir {path} is not a {vcs} repository")
-                continue
+            elif cylc.flow.flags.verbosity > -1:
```

Comment:
This seems to deal with the problem.

In my head there is a question about why we handle some SVN exc and not others - Why is `svn: warning: W155007` more worthy of capture and re-interpretation than `svn: warning: W155010`? I'm happy that this question only tangentially related to this PR.